src/bgen/Generator.NET.cs

```diff
@@ -0,0 +1,87 @@
+#if NET
```

Comment:
I think I'd keep both legacy and .NET in the same file/function as before and using a different filename (something like `Generator.PrintAttributes.cs`) and not two different files, it makes it easier to see that the logic is mostly for .NET, and it will also make the filenames make more sense once legacy is gone.

Reply:
Ugh I missed this occurrence, fixing

Reply:
👍 after this is addressed :)